source/Makefile

```diff
@@ -122,8 +128,14 @@ ${OBJ_DIR}/%.o:%.cpp
 ${OBJ_DIR}/%.o:%.f
 	${FORTRAN} -c ${HONG} $< -o $@	 
 
+${OBJ_DIR_serial}/%.o:%.cpp
```

Comment:
What's the difference between `OBJ_DIR_serial` and `OBJ_DIR`? From source file list to downstream compiling commands, they are identical.

Reply:
OBJ_DIR uses CPLUSPLUS_MPI to compile and OBJ_DIR_serial uses CPLUSPLUS to compile.

Reply:
> OBJ_DIR uses CPLUSPLUS_MPI to compile and OBJ_DIR_serial uses CPLUSPLUS to compile.

However, the source files between `FP_OBJS` and `FP_OBJS_serial` are of same(am I right?). I suggest using `${CPLUSPLUS}` with `${FP_OBJS}` but not `${FP_OBJS_serial}` in `serial` is enough, and no need to add `FP_OBJS_serial`. What do you think?